.gitignore

```diff
@@ -14,3 +14,5 @@ yarn-error.log
 
 # Ignore Dropbox files
 .DS_Store
+content/events/2020-amsterdam/.speakers.md.swp
```

Comment:
Please delete this file rather than adding it to the gitignore

Reply:
Done. Thank you